lib/materials/data_helpers.rb

```diff
@@ -111,6 +137,7 @@ def materials_data(materials, assigned_to_class = nil)
 
           subject_areas:    tags['subject_areas'],
           grade_levels:     tags['grade_levels'],
+          projects:         projects,
```

Comment:
I think it is in the model. It is used by the projects api controller: http://github.com/concord-consortium/rigse/blob/d3b960230784f3b4f19650bd28e9eb62b89ef92f/app/controllers/api/v1/projects_controller.rb#L11